consent/types.go

```diff
@@ -265,6 +265,10 @@ type HandledLoginRequest struct {
 	// authorization will be remembered for the duration of the browser session (using a session cookie).
 	RememberFor int `json:"remember_for"`
 
+	// RefreshRememberFor, if set to true, session cookie expiry time will be updated when session is
+	// refreshed (login skip=true).
+	RefreshRememberFor bool `json:"refresh_remember_for"`
```

Comment:
To be consistent with the existing values, it is better to name this property `refresh_remember` since it is a boolean value.

Existing props are following this format:
- `remember` - boolean
- `remember_for` - number

Reply:
Agree, maybe even clearer `extend_session_lifespan`. Could also be a time value alternatively (although it would be conflicting with `remember_for`)

Reply:
I'm on vacation and will come back to this in few weeks! :)